src/components/forms/form-elements/FormElements.vue

```diff
@@ -136,35 +136,56 @@
         <va-card :title="$t('forms.selects.title')">
```

Comment:
![image](https://user-images.githubusercontent.com/5394573/61462972-3747c500-a97c-11e9-8632-584f9fbf13fa.png)

Empty line should have still have some height.